src/plugins/ui_actions/public/cell_actions/mocks/helpers.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const makeAction = (actionsName: string, icon: string = 'icon', order: number = 0) => ({
```

Comment:
`order` is optional in the `Action` class. We should test what happens if it is not defined.